doc/api/url.md

```diff
@@ -1076,17 +1107,32 @@ added: v10.12.0
 This function ensures that `path` is resolved absolutely, and that the URL
 control characters are correctly encoded when converting into a File URL.
 
-```js
-new URL(__filename);                // Incorrect: throws (POSIX)
-new URL(__filename);                // Incorrect: C:\... (Windows)
-pathToFileURL(__filename);          // Correct:   file:///... (POSIX)
-pathToFileURL(__filename);          // Correct:   file:///C:/... (Windows)
+```mjs
+import url from 'url';
+new URL(__filename);                  // Incorrect: throws (POSIX)
+new URL(__filename);                  // Incorrect: C:\... (Windows)
+url.pathToFileURL(__filename);        // Correct:   file:///... (POSIX)
+url.pathToFileURL(__filename);        // Correct:   file:///C:/... (Windows)
```

Comment:
`__filename` is not defined in ESM, I don't think it makes sense to keep these examples.
```suggestion
```